HealthcareSystem/GraphicalEditorServerTests/DataFactory/ICreateTestObjectFactory.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GraphicalEditorServerTests.DataFactory
+{
+    public interface ICreateTestObjectFactory<T>
+    {
+        T CreateValidTestObject();
+        T CreateValidTestObject(int id);
+        T CreateInvalidTestObject();
+    }
```

Comment:
We had something like this interface in the PatientWebApp tests, but in the end we decided it was too restrictive and wasn't bringing any value and got rid of it.

Reply:
Alright, l have removed that part in last commit